src/lib/unsigned_extended/unsigned_extended.ml

```diff
@@ -176,3 +156,27 @@ module UInt32 =
       let of_uint64 =
         Fn.compose Unsigned.UInt32.of_int64 Unsigned.UInt64.to_int64
     end)
+
+(* since we don't have real versioning, check that serialization don't change *)
+let%test_module "Unsigned serializations" =
+  ( module struct
+    let%test "uint32 serialization" =
+      let uint32 = UInt32.of_int 9775 in
```

Comment:
Looks good. Can you dedup this test and the next one?

Reply:
do you mean, factor out common code?